assets/js/flexsearch.js

```diff
@@ -172,8 +172,9 @@ document.addEventListener("DOMContentLoaded", function () {
 
   // Preload the search index.
   async function preloadIndex() {
+    const tokenize = window?.hextra?.search?.flexsearch?.tokenize ?? 'forward'
```

Comment:
we could use hugo template here

for example: https://github.com/loenvom/hextra/blob/feat/custom_flex_search_tokenize/assets/js/flexsearch.js#L20